server.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const express = require('express');
+const bodyParser = require('body-parser');
+const Request  = require('request');
+const Querystring  = require('querystring');
+var cors = require('cors');
+const app = express();
+
+app.use(bodyParser.json());
+app.use(cors());
+
+//var csrf_guid = Guid.raw();
+var csrf_guid = 'test';
+const account_kit_api_version = 'v1.1';
+const app_id = '248317325862187';
+const app_secret = 'b50a135544538e8160de917d6d346160';
```

Comment:
You shouldn't commit a secret into git :) Use something like https://github.com/kevinsimper/hackyourfuture.dk/blob/master/backend/app.js#L1